package.json

```diff
@@ -23,7 +23,8 @@
     "start:mock-api": "cd node_modules/@manageiq/manageiq-api-mock/ && node index.js",
     "start:mock-webserver": "cd server && node app.js",
     "start:all": "yarn start:mock-api & yarn start:mock-webserver",
-    "test": "yarn vet && yarn gettext:extract && yarn gettext:validate && karma start karma.conf.js --single-run",
+    "test": "yarn vet && yarn gettext:extract && yarn gettext:validate && karma start karma.conf.js --single-run && yarn test:jest",
+    "test:jest": "jest .*.jest.spec.js",
```

Comment:
This is a nice way to integrate Jest into existing test toolchain, it allows you to gradually move tests over to (executed by) Jest.

Reply:
/cc @himdel